package.json

```diff
@@ -79,6 +78,8 @@
   },
   "devDependencies": {
     "autoprefixer": "^9.7.4",
+    "cpy-cli": "^3.1.1",
+    "cross-env": "^7.0.2",
```

Comment:
cross-env package for setting up cross-env environment variables and running some scripts.

Reply:
I went with cross-env @jhuynh85.